uk.co.compucorp.civicrm.tasksassignments/js/test/crm-tasks-workflows/controllers/casetype.extended.controller.spec.js

```diff
@@ -111,6 +119,44 @@
           });
         }
       });
+
+      describe('default case type category value', function () {
+        var CATEGORY_FIELD = 'custom_' + CASE_TYPE_CATEGORY_FIELD_ID;
+
+        describe('when creating a new case type', function () {
+          beforeEach(function () {
+            var caseType = getMockCaseType();
+            $scope = $rootScope.$new();
+
+            initController($scope, {
+              caseType: caseType
+            });
+          });
+
+          it('sets the default case type category equal to Workflow', function () {
+            expect($scope.caseType[CATEGORY_FIELD]).toEqual('Workflow');
+          });
+        });
+
+        describe('when updating an existing case type', function () {
+          var expectedCategory = 'Vacancy';
+
+          beforeEach(function () {
+            var caseType = getMockCaseType();
+            $scope = $rootScope.$new();
+            caseType.id = 100;
+            caseType[CATEGORY_FIELD] = expectedCategory;
+
+            initController($scope, {
+              caseType: caseType
+            });
+          });
+
+          it('does not override their current category', function () {
```

Comment:
I dont see this implemented any where. Am I missing something?

Reply:
it is implemented: https://github.com/compucorp/civihr-tasks-assignments/blob/d303ef7cd0d1a9c0c8be7af532b4b02d84856a2d/uk.co.compucorp.civicrm.tasksassignments/js/src/crm-tasks-workflows/controllers/casetype.extended.controller.js#L30-L33

This checks that if the case type is new it sets the default category as 'Workflow' (or to whatever is set as default from the API), and it would leave it as it is for existing case types (which is what this specific test is testing).